Tests/OpenTelemetrySdkTests/Logs/BatchLogRecordProcessorTests.swift

```diff
@@ -45,7 +45,7 @@ class BatchLogRecordProcessorTests : XCTestCase {
         let waitingExporter = WaitingLogRecordExporter(numberToWaitFor: maxQueueSize)
         let processor = BatchLogRecordProcessor(logRecordExporter: waitingExporter,scheduleDelay: 10000,maxQueueSize: maxQueueSize, maxExportBatchSize: maxQueueSize/2)
         let loggerProvider = LoggerProviderBuilder().with(processors: [processor]).build()
-        let logger = loggerProvider.get(instrumentationScopeName: "BatchLogRecordProcessorTest")
+        _ = loggerProvider.get(instrumentationScopeName: "BatchLogRecordProcessorTest")
```

Comment:
This is a valid warning, the test is not testing anything. The way to solve it is doing the actual test

Reply:
I saw the unfinished test, but decided to not fix it. As a compromise I replace the misleading warning with an explicit one, stating the test as unfinished.

https://github.com/open-telemetry/opentelemetry-swift/pull/424/commits/fb462d519d066841815a8dcf9d2821cd916c1911